src/app/feedback/feedback.component.ts

```diff
@@ -0,0 +1,56 @@
+import { Component, OnInit } from '@angular/core';
+import { UserService } from '../shared/user.service';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule }   from '@angular/forms';
+import { CouchService } from '../shared/couchdb.service';
+
+export class Feedback{
+  name:string
+  isUrgent:boolean;
+  feedbackType:string;
+  feedbackMsg:string;
+}
+
+@Component({
+  selector: 'feedback',
+  templateUrl: './feedback.component.html'
+})
+
+export class FeedbackComponent implements OnInit {
+  msgForUsr:string;
+  isFeedbackMsg :boolean=true;
+  feedback:Feedback= new Feedback();
+  message:string;
+  fedbkSubmitted:boolean= false;
+
+  constructor(
+    private userService: UserService,
+    private couchService:CouchService
+    ) { }
+
+  ngOnInit() {
+    this.feedback.name= this.userService.get().name;
+  }
+
+  submitfeedback(){
+    if (this.feedback.feedbackMsg===undefined|| this.feedback.feedbackMsg=="") {
+      this. msgForUsr="Feedback  cannot be empty";
+      this.isFeedbackMsg=false;
+    }else
+    {
+      this.isFeedbackMsg=true;
+      this.couchService.post('feedback/', this.feedback)
+      .then((data) => {
+        this. msgForUsr='';
```

Comment:
There should not be a space between the `this` and the variable name. Please make sure to run the ng lint either with the pre push hook or from your terminal and fix the errors it finds.

Reply:
this.msgForUsr is used twice in the code without any case to justify. It could just be used once in this case.